misc/userscripts/qute-lastpass

```diff
@@ -68,6 +69,20 @@ group.add_argument('--password-only', '-w',
 
 stderr = functools.partial(print, file=sys.stderr)
 
+# username insertion JS code
+# Apparently Github.com's email login box' type is not "email", but "text"
+# Its name is "login"
```

Comment:
I'm personally not a fan of this method because it makes it completely impossible to enter the username when the username field dosen't match the hardcoded patterns. Also, if a match is falsefully found (for example, there is an earlier element that contains 'user') that element will get the text instead of the selected one.

The existing method does work for all sites though, using the flags for username/password only (if `<tab>` will not work to go between the fields as described in the issue you filed, you can use the username only flag, then select the password field, and then the password only flag). It also allows the user to have much more control, for example, I can insert my username into the textbox I am typing this comment in using the username only flag, or I can insert my password into non-password designated text fields.

The two methods probably need more thought. Maybe we should add a library to handle actually filling the passwords (and switching between methods) so the userscripts don't have to worry about it at all, especially since we have multiple password filling userscripts.

Do keep in mind that most of the other password scripts (qute-pass, qute-keepass) currently use the typing method, and password_fill uses the js method.